web/src/components/molecules/AccountSettings/ServiceForm.tsx

```diff
@@ -0,0 +1,65 @@
+import { useCallback, useMemo } from "react";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Form from "@reearth-cms/components/atoms/Form";
+import Select from "@reearth-cms/components/atoms/Select";
+import { User } from "@reearth-cms/components/molecules/AccountSettings/types";
+import { localesWithLabel, useT } from "@reearth-cms/i18n";
+
+export type Props = {
+  user?: User;
+  onLanguageUpdate: (lang?: string | undefined) => Promise<void>;
+};
+
+const AccountServiceForm: React.FC<Props> = ({ user, onLanguageUpdate }) => {
+  const [form] = Form.useForm();
+  const { Option } = Select;
+  const t = useT();
+
+  const langItems = useMemo(
+    () => [
+      { key: "und", label: t("Auto") },
+      ...Object.keys(localesWithLabel).map(l => ({
+        key: l as keyof typeof localesWithLabel,
+        label: localesWithLabel[l as keyof typeof localesWithLabel],
+      })),
+    ],
+    [t],
+  );
+
+  const handleSubmit = useCallback(async () => {
+    try {
+      const values = await form.validateFields();
+      await onLanguageUpdate?.(values.lang);
+    } catch (info) {
+      console.log("Validate Failed:", info);
+    }
```

Comment:
Same as elsewhere I commented, remove the try/catch if not needed

Reply:
@KaWaite , updated, same reason as I stated above as well.